.github/workflows/ci.yml

```
@@ -10,13 +10,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        include:
-          - python-version: "3.7"
-            nautobot-version: "1.2"
-          - python-version: "3.8"
-            nautobot-version: "1.3"
-          - python-version: "3.9"
-            nautobot-version: "1.3"
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        nautobot-version: ["1.2", "1.3", "1.4"]
```

Looks like we need to exclude a combination that doesn't exist.

```suggestion
        nautobot-version: ["1.2", "1.3", "1.4"]
        exclude:
          - python-version: 1.2
            nautobot-version: 3.10
```